packages/boot/test/orchestration/axelar-gmp.test.ts

```diff
@@ -0,0 +1,445 @@
+    wallet,
+    previousOffer: previousOfferId,
+    methodName: 'sendGmp',
+    offerArgs: [
+      {
+        destinationAddress: '0x20E68F6c276AC6E297aC46c84Ab260928276691D',
+        type: 1,
+        gasAmount: 20000,
+        destinationEVMChain: 'Ethereum',
+        contractInvocationData: {
+          functionSelector: utils.id('setCount(uint256)').slice(0, 10),
+          deadline: 5000,
+          nonce: 7,
+          encodedArgs,
+        },
+      },
+    ],
+    proposal: {
+      give: { BLD: { brand: BLD, value: 1n } },
+    },
+  });
+
+  t.deepEqual(getLogged(), [
+    'Inside sendGmp',
+    'Payload: [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,32,230,143,108,39,106,198,226,151,172,70,200,74,178,96,146,130,118,105,29,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,7,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,19,136,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,160,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,36,209,78,98,184,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,234,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]',
+    'Fee object {"amount":"20000","recipient":"axelar1zl3rxpp70lmte2xr6c4lgske2fyuj3hupcsvcd"}',
+    'Initiating IBC Transfer...',
+    'DENOM of token:ubld',
+    'sendGmp successful',
+  ]);
+
+  t.like(wallet.getLatestUpdateRecord(), {
+    status: {
+      id: `evmTransaction${evmTransactionCounter - 1}`,
+      result: 'sendGmp successful',
+    },
+  });
+
+  t.log('make offer without contractInvocationData');
+  t.context.storage.data.delete('published.axelarGmp.log');
+
+  await makeEVMTransaction({
+    wallet,
+    previousOffer: previousOfferId,
+    methodName: 'sendGmp',
+    offerArgs: [
+      {
+        destinationAddress: '0x20E68F6c276AC6E297aC46c84Ab260928276691D',
+        type: 1,
+        gasAmount: 20000,
+        destinationEVMChain: 'Ethereum',
+      },
+    ],
+    proposal: {
+      give: { BLD: { brand: BLD, value: 1n } },
+    },
+  }).catch(_err => {
+    t.like(wallet.getLatestUpdateRecord(), {
+      status: {
+        id: `evmTransaction${evmTransactionCounter - 1}`,
+        error: 'Error: contractInvocationData is not defined',
+      },
+    });
+  });
+
+  t.log('make offer without passing gas amount');
+  t.context.storage.data.delete('published.axelarGmp.log');
+
+  await makeEVMTransaction({
+    wallet,
+    previousOffer: previousOfferId,
+    methodName: 'sendGmp',
+    offerArgs: [
+      {
+        destinationAddress: '0x20E68F6c276AC6E297aC46c84Ab260928276691D',
+        type: 1,
+        destinationEVMChain: 'Ethereum',
+        contractInvocationData: {
+          functionSelector: utils.id('setCount(uint256)').slice(0, 10),
+          deadline: 5000,
+          nonce: 7,
+          encodedArgs,
+        },
+      },
+    ],
+    proposal: {
+      give: { BLD: { brand: BLD, value: 1n } },
+    },
+  }).catch(_err => {
```

Comment:
We typically use `await t.throwsAsync()` to verify errors, but these look good.

Also - consider using `test.serial` and storing things in test context instead of having one giant test. It helps with debugging and maintenance